app/assets/stylesheets/hackney.scss

```diff
@@ -21,15 +21,25 @@ header {
 
   h1 {
     background: asset-url('logo.png') 0 0 no-repeat;
+    display: inline-block;
     height: 54px;
     overflow: hidden;
     text-indent: 100%;
+    vertical-align: middle;
     white-space: nowrap;
     width: 300px;
   }
 
   a {
     color: $white;
+    text-decoration: none;
+  }
+
+  h2 {
+    @include heading-24;
+    display: inline-block;
+    margin-left: 1em;
```

Comment:
Consider removing the margin at breakpoints where the service description wraps, so that it lines up with the logo: <img width="449" alt="screen shot 2017-10-02 at 11 34 44" src="https://user-images.githubusercontent.com/393167/31073893-e81baf42-a765-11e7-83e6-debe373c9d81.png">

Reply:
Good spot :+1:

I've added a fix for this and amended the original commit.

<img width="342" alt="mobile-header" src="https://user-images.githubusercontent.com/3166/31074600-4e7d08aa-a769-11e7-822f-51aa50d62edd.png">